### ISSUE TITLE
About LESS, CSS, minified CSS files nesting

### ISSUE THREAD

**Issue by Azaret** [madskristensen/WebEssentials2013#611]
Hi,

I just updated Web Essentials 2013 to 1.8, and right away something is bothering me.
As related to #545 I use LESS file in some of my project, but only use minified CSS files for Web Deploy. I don't need more and don't want more, having CSS files is just a waste, and there is enough files in huge project with dozen of style sheets or more. I honestly don't like to have those file 3 or 4 times (4 because you enabled gzip by default in 1.8, no thanks).

Anyway while waiting for #545 to be solved in anyway, I was just deleting manually CSS files when I updated the LESS files. It was boring but a problem. But now it is, because with this nesting thing, I can't delete those useless files anymore without having to play with the csproj file. More that, since one css.gzip is linked to just one min.css which is linked to just one .css which is linked to just one less file, nesting those is poor design.

Tl;dr: So, is this nesting thing could be avoided please ?

Thanks
